services/director-v2/src/simcore_service_director_v2/api/routes/computations_tasks.py

```diff
@@ -164,22 +168,28 @@ async def get_task_log_file(
     return await _get_task_log_file(user_id, project_id, node_uuid)
 
 
-# NOTE: This handler function is NOT ACTIVE
-# but still kept as reference for future extensions that will tackle
-# real-time log streaming (instead of logfile download)
-#
-# @router.get(
-#    "/{project_id}/tasks/{node_uuid}/logs",
-#    summary="Gets computation task log",
-# )
-# - Implement close as possible to https://docs.docker.com/engine/api/v1.41/#operation/ContainerTop
-async def get_task_logs(
-    user_id: UserID,
+@router.post(
+    "/{project_id}/tasks/-/outputs:batchGet",
+    summary="Gets all outputs for selected tasks",
+    response_model=TasksOutputs,
+    responses={
+        status.HTTP_404_NOT_FOUND: {
+            "description": "Cannot find computation or the tasks in it"
+        }
+    },
+)
```

Comment:
I would guess there could also be some globally handled exceptions that can be raised inside `comp_tasks_repo.get_outputs_from_tasks`, but I guess it is not crucial at this stage to handle these.